java_exception_test.go

```diff
@@ -19,14 +19,12 @@ import (
 	"testing"
 )
 
-func TestThrowable(t *testing.T) {
-	testDecodeFrameworkFunc(t, "throw_throwable", func(r interface{}) {
-		assert.Equal(t, "exception", r.(error).Error())
-	})
-}
+func TestThrowable(t *testing.T) { doTest(t, "throw_throwable", "exception") }
```

Comment:
only need to keep one test func `TestException`, which contains all calling of `doTestException`.

Reply:
> only need to keep one test func `TestException`, which contains all calling of `doTestException`.

ok